hms-app/src/components/PitchTimer.tsx

```diff
@@ -42,19 +50,57 @@ const useStyles = createStyles((_theme, _params, getRef) => ({
   finished: {
     color: _theme.colors.red,
   },
+
+  fullscreenOverlay: {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    width: '100vw',
+    height: '100vh',
+    zIndex: 2,
+  },
+
+  bgRed: {
+    background: 'radial-gradient(rgba(255,52,52,0.3), rgba(255,52,52,1))',
+  },
+
+  bgOrange: {
+    background: 'radial-gradient(rgba(255,125,25,0.3), rgba(255,125,25,1))',
+  },
+
+  fullscreenCounter: {
+    fontSize: 150,
+    fontWeight: 'bold',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+  },
+
+  progress: {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    width: '100%',
+  },
 }))
 
 const getMillis = (minutes: number, seconds: number) => {
   return minutes * 60 * 1000 + seconds * 1000
 }
 
+const getSeconds = (millis: number) => {
+  return Math.floor(millis / 1000)
+}
+
 function PitchTimer(time: PitchTimerProps) {
```

Comment:
`time` isn't really a good name for this variable.  Just `props` would be better (since it does contain more than time information).